### DISCUSSION TITLE
Reminding beginners about BitLocker encrypttion before installing Ubuntu

### DISCUSSION THREAD

**Discussion by mshafeeksa** [TheOdinProject/curriculum#26928] [General]
### Backstory:
This suggestion comes from a bitter experience of mine. I started TOP few days back and had to install Ubuntu to my laptop. I followed the steps given in the course and after installing Ubuntu, I realized that there was a Bitlocker encryption on my drives which disallowed me to login to my PC. It asked me to go back to Windows and remove the encryption. I restarted my PC to realize that I had to enter a **Bitlocker Encryption recovery key** to login to Windows. I had no idea what Bitlocker Encryption is and had to do some research on internet about it. In my research, I found that some new laptops come with a Bitlocker Encryption done by manufacturer themselves. And the recovery key is in the Microsoft account of the user. To my bad luck, when I went to the link given by Microsoft forums to find the key, it said 'Try a different URL'. On further research, I realized that this is a known issue for many years, many users have been affected by it and Microsoft have not addressed this issue yet. **Long story short, I had to format my entire PC and lose all the files I had.**
### Suggestions:
It would be great to warn users about the Bitlocker Encryption. Encouraging users to verify whether their PC is encrypted, and if so, advising them to securely save or print their Bitlocker Encryption recovery key before installing Ubuntu, is a proactive step to prevent initial pitfalls in this exciting journey with a positive outlook.
Cheers

**Comment by CouchofTomato** [comment]
@xandora Do you have an opinion on this mate?

**Reply by xandora** [reply]
@CouchofTomato I have several opinions on this. I think the end result is that we need to totally replace the current dual-boot guide that we're offering. It causes many other issues that I'm not super thrilled about.

I agree that we need something that mentions Bitlocker, but it probably needs to be part of the replacement guide, and not *yet another* NOTE section in our curriculum. The installations section is getting bogged down heavily with the gotcha sections 😠
